src/main/java/ladder/domain/ExecutionResultMap.java

```diff
@@ -0,0 +1,50 @@
+package ladder.domain;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collector;
+import java.util.stream.IntStream;
+
+public class ExecutionResultMap {
+    private Map<String, String> executionResultMap;
+
+    private ExecutionResultMap() {
+        this(new LinkedHashMap<>());
+    }
+
+    private ExecutionResultMap(final Map<String, String> executionResultMap) {
+        this.executionResultMap = executionResultMap;
+    }
+
+    public static ExecutionResultMap of(Persons persons, LadderExecutionResults ladderExecutionResults) {
+        return IntStream.range(0, persons.getCountOfPerson())
+                .boxed()
+                .collect(Collector.of(
+                        ExecutionResultMap::new,
+                        (executionResultMap, index) ->
+                                executionResultMap.add(
+                                        persons.getPerson(index),
+                                        ladderExecutionResults.getExecutionResult(index)),
+                        (a, b) -> {
+                            throw new UnsupportedOperationException();
+                        }
```

Comment:
```suggestion
                        nope()
```
이부분 역시 Util 클래스를 만들어 static import 를 활용해 볼 수 있을것 같습니다.
```java
public class BinaryOperations {
    public static <T> BinaryOperator<T> nope() {
        return (a, b) -> {
            throw new UnsupportedOperationException();
        };
    }
}
```